engines/director/lingo/lingo-object.cpp

```diff
@@ -907,8 +910,21 @@ bool BitmapCastMember::setField(int field, const Datum &d) {
 		warning("STUB: BitmapCastMember::setField(): Unprocessed setting field \"%s\" of cast %d", g_lingo->field2str(field), _castId);
 		return false;
 	case kTheRegPoint:
-		warning("STUB: BitmapCastMember::setField(): Unprocessed setting field \"%s\" of cast %d", g_lingo->field2str(field), _castId);
-		return false;
+		if (d.type == POINT || (d.type == ARRAY && d.u.farr->arr.size() == 2)) {
```

Comment:
Did you test it with arrays? Also, did you test it with arrays _less_ than 2 and _more_ than 2 items?

Other tests show that when there are MORE items, it takes the first two and treats them as POINT